models/balancer/arbitrum/balancer_v2_arbitrum_protocol_fee.sql

```diff
@@ -4,7 +4,7 @@
     config(
         schema = 'balancer_v2_arbitrum',
         alias = 'protocol_fee', 
-        materialized = 'incremental',
+        materialized = 'table',
```

Comment:
think a typo to leave only arbitrum as `table`?

Reply:
my bad. fixed.